cinema/migrations/0003_movie_duration.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cinema', '0002_initial'),
+        ("cinema", "0002_initial"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='movie',
-            name='duration',
+            model_name="movie",
+            name="duration",
             field=models.IntegerField(default=123),
```

Comment:
The `default` value is set to `123`, but `preserve_default` is set to `False`. This means that the default value will not be preserved for existing rows in the database. If you intend to apply this default value to existing records, consider setting `preserve_default` to `True` or handling existing records separately.